### PR TITLE
improvement/Button: Adds Button and OutlineButton

### PR DESCRIPTION
**Pull request by willthebrit**

### CODE REVIEW

File: packages/button/src/Button.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import BaseButton from './BaseButton'
```
Comment (billyxs) [inline packages/button/src/Button.js]: With the default Button export, do we want to allow an `outline` prop to be passed in to toggle which kind of button is being used, or are we set on requiring an explicit import of OutlineButton?

**Comment by seesleestak** [comment]
Button has never looked better :)

**Comment by seesleestak** [comment]
One thing I noticed on the hui-pages documentation is that the default outline button has dark font color while the border is light. Whereas with the other outline buttons, the font color and border color is the same.

File: packages/hui-pages/src/routes/ButtonDocs/index.js [hunk 2]
```diff
@@ -1,119 +1,82 @@
 import React from 'react'
-import { Button, OutlineButton } from '@hixme-ui/button'
+import { Grid, Row, Col } from 'react-flexbox-grid'
+import Button from '@hixme-ui/button'
 
 import Text from '@hixme-ui/text'
```
Comment (seesleestak) [inline packages/hui-pages/src/routes/ButtonDocs/index.js]: I get a lint error on this cause it looks like its not getting used

**Pull request merged**